pkg/config/config_test.go

```diff
@@ -70,3 +68,68 @@ func TestSingleStringPairs(t *testing.T) {
 
 	r.Equal(c1, c2)
 }
+
+// A DummySource represents a source that generates random data.
+type ListSource struct {
+	BulletPoints []string
+}
+
+// Define Read function
+func (listSrc *ListSource) Read() (map[string]interface{}, error) {
+	dummyIface := make(map[string]interface{})
+	dummyIface[source.Secret] = []string{"item1", "item2", "item3"}
+	return dummyIface, nil
+}
+
+// Kind returns the kind of this source
+func (listSrc *ListSource) Kind() source.Kind {
+	return source.Kind("listSource")
+}
+
+// Todo: Think about whether the dummy source suffices
+func TestConfigWithLists(t *testing.T) {
```

Comment:
This test currently fails this way:
```
--- FAIL: TestConfigWithLists (0.00s)
    /Users/anneku/Code/7/rotator/pkg/config/config_test.go:105: 
        	Error Trace:	config_test.go:105
        	Error:      	Expected nil, but got: Unrecognized source
        	Test:       	TestConfigWithLists
FAIL
FAIL	github.com/chanzuckerberg/rotator/pkg/config	1.589s
FAIL
```
Seems like I'd have to add `ListSource` support in secret.MarshalYAML() to keep going